src/asset_util/src/lib.rs

```diff
@@ -29,13 +29,16 @@ pub const IC_CERTIFICATE_EXPRESSION: &str =
 /// for the certification to be valid.
 #[derive(Debug, Default, Clone)]
 pub struct CertifiedAssets {
-    assets: HashMap<String, (Vec<HeaderField>, Vec<u8>)>,
+    /// Map from URL path to tuple of response (status_code, headers, body)
+    assets: HashMap<String, (u16, Vec<HeaderField>, Vec<u8>)>,
```

Comment:
nit: Why don't we use `HashMap<string, CertifiedAssed` here?

Reply:
Ah yes, of course. 👍 
This now also finally resolves the issue of `CertifiedAssets` not containing `CertifiedAssed`. 😉